### PR TITLE
PYIC-7470: Output report in logs

### PR DESCRIPTION
**Pull request by Wynndow**
### What changed

Output report in logs

### Why did it change

If for some reason we fail to catch the output of the lambda invocation we still want to be able to retrieve the report.

### Issue tracking

- [PYIC-7470](https://govukverify.atlassian.net/browse/PYIC-7470)

[PYIC-7470]: https://govukverify.atlassian.net/browse/PYIC-7470?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java [hunk 1]
```diff
@@ -220,6 +226,7 @@ public BatchReport handleRequest(Request request, Context context) {
             forkJoinPool.shutdown();
         }
 
+        logReport(report);
```
Comment (Joe-Edwards-GDS) [inline lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java]: If you put this in the `finally` block, then you don't need to repeat it - and it's clearer what the intent is IMO

**Review by Joe-Edwards-GDS — DISMISSED** [review]
The report only includes hashed user IDs right (i.e. nothing potentially sensitive)? Seems like a sensible thing to do.

One small suggestion...

File: lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java (hunk 1 above)
Comment (Wynndow) [reply lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java]: I think that's a good shout - I'll shift it

Comment (Wynndow) [reply lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java]: And yeah - just hashes of user IDs

**Review by Joe-Edwards-GDS — APPROVED** [review]

**Pull request merged**